src/main/resources/db/changelog/changes/10-create-orders-tables.yaml

```diff
@@ -0,0 +1,81 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-orders-table
+      author: sonia_baranova
+      changes:
+        - createTable:
+            tableName: orders
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: user_id
+                  type: BIGINT
+                  constraints:
+                    foreignKeyName: fk_order_user
+                    references: users(id)
+              - column:
+                  name: status
+                  type:  ENUM('PENDING', 'PROCESSING', 'SHIPPED', 'DELIVERED', 'CANCELED')

```

Comment:
I don’t  recommend doing it because you can add a list of statuses  in the future